Mobile Buy SDK/Mobile Buy SDK/Data/BUYClient+Storefront.m

```diff
@@ -173,6 +173,27 @@ - (NSOperation *)getCollectionsPage:(NSUInteger)page completion:(BUYDataCollecti
 	}];
 }
 
+- (NSOperation *)getCollectionsByIds:(NSArray<NSString *> *)collectionIds completion:(BUYDataCollectionsBlock)block
+{
+	return [self getCollectionsByIds:collectionIds page:0 completion:block];
```

Comment:
> I don't think so, I believe asking for page 0 == page 1

Yup 👍 

@gabrieloc, are you seeing a different behaviour because you're not specifying a `limit`?